recommendation-app/README.md

```diff
@@ -57,4 +57,24 @@ The steps in the [README.md](../README.md) required to be completed for setting
    ```
 The expected format of the result is `userId`, `comma separated 6 product ids` and `timestamp` of the window.
 
+You can also deploy Prometheus to monitor the metrics inside job manager and task manager.
+5. Update the prometheus configuration with the ip address of flink pods to scrape from:
+   ```
+   sed -i s/JOB_MANAGER/$(kubectl get pods -lapp=recommendation-app -o=jsonpath="{range .items[*]}{.status.podIP}{','}{end}" | cut -d ',' -f1)/g recommendation-app/prometheus-install/prometheus-config.yaml
+   sed -i s/TASK_MANAGER/$(kubectl get pods -lapp=recommendation-app -o=jsonpath="{range .items[*]}{.status.podIP}{','}{end}" | cut -d ',' -f2)/g recommendation-app/prometheus-install/prometheus-config.yaml
+   ```
```

Comment:
These commands presumably work fine on linux, but not for Mac. So we should list these for linux and these for Mac:

```
sed -i '' s/JOB_MANAGER/$(kubectl get pods -lapp=recommendation-app -n flink -o=jsonpath="{range .items[*]}{.status.podIP}{','}{end}" | cut -d ',' -f1)/g recommendation-app/prometheus-install/prometheus-config.yaml
sed -i '' s/TASK_MANAGER/$(kubectl get pods -lapp=recommendation-app -n flink -o=jsonpath="{range .items[*]}{.status.podIP}{','}{end}" | cut -d ',' -f2)/g recommendation-app/prometheus-install/prometheus-config.yaml
```

Reply:
Ah, right, I was testing in Ubuntu. Added MacOS version. Thanks.